benchmarks/BM_mathfuns.cpp

```diff
@@ -133,6 +135,29 @@ BENCHMARK_DEFINE_F(MyFixture, SIMDCos)
     }
 }
 
+BENCHMARK_DEFINE_F(MyFixture, ScalarLibmFloorLog2)
+(benchmark::State& state)
+{
+    for (auto _ : state) {
+        for (size_t i = 0, n = source.size(); i < n; ++i) {
+            intResult[i] = static_cast<int>(
+                std::floor(std::log2(std::fabs(source[i]))));
+        }
+        benchmark::DoNotOptimize(intResult);
```

Comment:
Do you need this? If it's declared in the fixture I think it will assume there are side effects. I could be wrong though.

Reply:
Honestly, no idea. If you ask me why it's there, I copied the method of doing from other test cases above.

Reply:
Ok so it's because of past-me then!